pkg/platform/model/inventory.go

```diff
@@ -284,17 +293,17 @@ func filterPlatformIDs(hostPlatform, hostArch string, platformIDs []strfmt.UUID)
 	return nil, &ErrNoMatchingPlatform{hostPlatform, hostArch, libcVersion}
 }
 
-func fetchLibcVersion() string {
+func fetchLibcVersion() (string, error) {
 	if runtime.GOOS != "linux" {
-		return ""
+		return "", nil
 	}
 
-	libcInfo := sysinfo.GetRequestedLibcInfo()
-	if libcInfo == nil {
-		return ""
+	cfg, err := config.New()
```

Comment:
Is this really okay? Or do we need to be passing the config instance from main all the way to here?

Reply:
Updated to pass the config instance down from main.